### ISSUE TITLE
ng 10 Warning: Unable to fully load

### ISSUE THREAD

**Issue by sysmat** [ng-bootstrap/ng-bootstrap#3825]
### Bug description:
When building I got a lot of warnings

```
Warning: Unable to fully load /node_modules/@ng-bootstrap/ng-bootstrap/timepicker/timepicker-config.d.ts for source-map flattening: Circular source file mapping dependency: /node_modules/@ng-bootstrap/ng-bootstrap/timepicker/timepicker-config.d.ts.map
/node_modules/@ng-bootstrap/ng-bootstrap/timepicker/timepicker-config.d.ts.map
Warning: Unable to fully load /node_modules/@ng-bootstrap/ng-bootstrap/timepicker/ngb-time-adapter.d.ts for source-map flattening: Circular source file mapping dependency: /node_modules/@ng-bootstrap/ng-bootstrap/timepicker/ngb-time-adapter.d.ts.map -> /node_modules/@ng-bootstrap/ng-bootstrap/timepicker/ngb-time-adapter.d.ts.map

```
### Versions of Angular, ng-bootstrap and Bootstrap:

Angular: 10.0.8

ng-bootstrap: 7.0.0

Bootstrap: 4.5.1

**Comment by maxokorokov** [comment]
We would need reproduction steps for this, please. We haven't seen this issue before.

If you have a minimal Angular CLI project to clone (starting with empty an one, adding ng-bootstrap and reproducing the issue with source maps), it would be helpful. Otherwise not much we can do.

**Comment by benouat** [comment]
Closing as no feedback. Don't hesitate to re-open with proper information if needed

**Comment by nayfin** [comment]
This happened to me with a library I published when I consumed it in another app. All the components warned of a circular dependency on themselves. It looks like it was an issue with the compiler interacting with node_modules. Deleting `node_modules` and `package-lock.json` then verify cache and reinstalling seems to have fixed it.  

`rm -rf node_modules`
`rm package-lock.json`
`npm cache verify`
`npm install`
